lib/cookie.js

```diff
@@ -40,7 +43,7 @@ module.exports = class Cookie {
   }
 
   set maxAge (ms) {
-    this.expires = new Date(Date.now() + ms)
+    if (!this.expires) { this.expires = new Date(Date.now() + ms) }
```

Comment:
We are not setting the maxAge any more here - why should we check it?

Reply:
This is if conduction ensures that `this.expiry` is only set for newly created session and not for existing session loaded from session store .  
For session loaded from session store, the `expiry` value will be already there ( it is determined at the time of session creation  ).